src/stream.ts

```diff
@@ -53,7 +53,7 @@ export class DataAndMoneyStream extends Duplex {
   protected bytesRead: number
 
   constructor (opts: StreamOpts) {
-    super()
+    super({ allowHalfOpen: false })
```

Comment:
Can you put a comment explaining that this is not currently implemented but may be added later?